core/execution/spot/market.go

```diff
@@ -273,9 +273,18 @@ func NewMarket(
 	}
 	liquidity.SetGetStaticPricesFunc(market.getBestStaticPricesDecimal)
 
+	book.SetOffbookSource(&offbook{})
 	return market, nil
 }
 
+// some dummy offbook source to stop the spot tests falling over, but we should revisit when we revive spots.
+type offbook struct{}
+
+func (ob *offbook) SubmitOrder(_ *types.Order, _, _ *num.Uint) []*types.Order {
+	return nil
+}
+func (ob *offbook) NotifyFinished() {}
+
```

Comment:
Spot are being revived now, but without AMM support. let's make sure this is good enough to run in production.

Reply:
It think it will be fine, there was a feature test that went through it that is still passing. We'll want to rejected `create-amm` transactions to spot markets anyway, so I'll make a ticket for that and to make sure nothing else weird might happen when spots are about.